docs/api-and-integration/metadata/jira.md

```diff
@@ -1,75 +1,122 @@
+
+    Note: Keep this configuration page open while you complete the next steps as you need to copy values between Octopus and Jira.
+
+2. Configure the Jira extension in Octopus Deploy.
+
+    In the Octopus web portal, navigate to **{{Configuration,Settings,Jira Issue Tracker}}** and copy the following values from the Jira App configuration page:
+
+    - **Jira Base URL**. This tells Octopus where your Jira instance is located and enables Octopus to render the links back to Jira issues. i.e., https://your-jira-instance.atlassian.net.
+    - **Jira Connect App Password**.
+
+    Ensure the **Is Enabled** property is set as well.
+
+3. In Octopus Deploy Configure the Release Note Options (optional).
+
+    - **Jira username/password**: Set these values to allow Octopus to connect to Jira and retrieve Jira issue (work item) details when viewing packages or creating releases. If these are not provided, just the raw work item references will be used as the work item link descriptions. If they are provided the work item's title will be used as the work item link's description.
+
+    The password should be an API Token, rather than an actual password. You can create an API token from an Atlassian account in the 'Security' area.
+    
+    - **Release Note Prefix**: If specified, Octopus will look for a comment that starts with the given prefix text and use whatever text appears after the prefix as the release note, which will come through to the [release notes templates](/docs/api-and-integration/metadata/release-notes-templates.md) etc as the work item link's description. If no comment is found with the prefix then Octopus will default back to using the title for that work item.
 
-On success the blocks will appear with green text just like the other blocks in the log.
+    For example, a prefix of `Release note:` can be used to identify a customer friendly issue title vs a technical feature or bug fix title.
 
-## Deployment Events
+4. Ensure the Octopus Server URL is set.
 
-The end result of this integration is to provide Jira with updates on the progress of work items through the pipeline. It will display information similar to the following when it receives these updates for the work items:
+    If you are using Octopus Cloud, this value is automatically set for you. If you are not using Octopus Cloud, navigate to the **{{Configuration,Nodes}}** page and ensure you have set the Server URI field to your Octopus Server's base URL. i.e., https://my-company.octopus.app or https://my-company-internal-name
+    Note: Octopus passes this value to Jira so it can build hyperlinks back to the deployments from its UI. It never actually tries to connect to this Url itself.
```

Comment:
```suggestion
    Note: Octopus passes this value to Jira so it can build hyperlinks back to the deployments from the Jira UI. It never actually tries to connect to this Url itself.
```
I think most people would have know what you meant, but for those few who'd have to backtrack and reread.